tests/Dockerfile

```diff
@@ -0,0 +1,36 @@
+# The Dockerfile's resulting image is purpose-built for executing Jaeger Operator e2e tests within the OpenShift release (https://github.com/openshift/release) using Prow CI.
+FROM golang:1.20
+
+# Set the user to root
+USER root
+
+# Copy the repository files
+COPY . /tmp/jaeger-operator
+
+WORKDIR /tmp
```

Comment:
This could be removed as it is set below to `WORKDIR /tmp/jaeger-operator`

Reply:
@pavolloffay removed that workdir.